### ISSUE TITLE
signal.convolve2d from scipy results in UnboundVariable errors

### ISSUE THREAD

**Issue by iutlu** [microsoft/pylance-release#488]
## Environment data

-   Language Server version: 2020.10.0
-   OS and version: macOS Catalina 10.15.4 remoting into Ubuntu 18.04.4 LTS
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8 (conda-forge)

## Expected behaviour

Return type for `signal.convolve2d` from `scipy` is `Unbound`, resulting spurious unbound variable errors.

## Actual behaviour

No spurious unbound variable errors. (See lines 11, 20 in the attached snippet.)

## Logs

```
User belongs to experiment group 'AlwaysDisplayTestExplorer - experiment'
User belongs to experiment group 'ShowPlayIcon - start'
User belongs to experiment group 'DebugAdapterFactory - experiment'
User belongs to experiment group 'PtvsdWheels37 - experiment'
User belongs to experiment group 'UseTerminalToGetActivatedEnvVars - control'
User belongs to experiment group 'AA_testing - control'
User belongs to experiment group 'LocalZMQKernel - experiment'
User belongs to experiment group 'CollectLSRequestTiming - control'
User belongs to experiment group 'CollectNodeLSRequestTiming - experiment'
User belongs to experiment group 'EnableIPyWidgets - experiment'
User belongs to experiment group 'RunByLine - experiment'
User belongs to experiment group 'CustomEditorSupport - control'
User belongs to experiment group 'pythonaa'
> conda --version
> pyenv root
> conda info --json
> python3.7 ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> python3.6 ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> python3 ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> python2 ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> python ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
> /mnt/pixery/miniconda3/envs/bug-report/bin/python ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py -c "import sys;print(sys.executable)"
Error 2020-10-14 11:04:55: Failed to check if file needs to be fixed [EntryNotFound (FileSystemError): Unable to read file 'vscode-remote://ssh-remote+ec2/home/ubuntu/.config/Code/User/settings.json' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open '/home/ubuntu/.config/Code/User/settings.json')
	at _handleError (/home/ubuntu/.vscode-server/bin/93c2f0fbf16c5a4b10e4d5f89737d9c2c25488a3/out/vs/server/remoteExtensionHostProcess.js:750:473)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async y.readText (/home/ubuntu/.vscode-server/extensions/ms-python.python-2020.9.114305/out/client/extension.js:1:503784)
	at async p.doesFileNeedToBeFixed (/home/ubuntu/.vscode-server/extensions/ms-python.python-2020.9.114305/out/client/extension.js:56:962033)
	at async /home/ubuntu/.vscode-server/extensions/ms-python.python-2020.9.114305/out/client/extension.js:56:961160
	at async Promise.all (index 0)
	at async p.getFilesToBeFixed (/home/ubuntu/.vscode-server/extensions/ms-python.python-2020.9.114305/out/client/extension.js:56:961106)
	at async p.updateTestSettings (/home/ubuntu/.vscode-server/extensions/ms-python.python-2020.9.114305/out/client/extension.js:56:960733)] {
  code: 'FileNotFound',
  name: 'EntryNotFound (FileSystemError)'
}
> conda env list
> conda env list
Starting Pylance language server.
Python interpreter path: /mnt/pixery/miniconda3/envs/bug-report/bin/python
> /mnt/pixery/miniconda3/envs/vscode-dev/bin/bandit --configfile /mnt/pixery/ihsan/bugreport/bandit.yaml -f custom --msg-template {line},0,{severity},{test_id}:{msg} -n -1 /mnt/pixery/ihsan/bugreport/test.py
cwd: /mnt/pixery/ihsan/bugreport
> /mnt/pixery/miniconda3/envs/vscode-dev/bin/flake8 --config=/mnt/pixery/ihsan/bugreport/.flake8 --format=%(row)d,%(col)d,%(code).1s,%(code)s:%(text)s /mnt/pixery/ihsan/bugreport/test.py
cwd: /mnt/pixery/ihsan/bugreport
> /mnt/pixery/miniconda3/envs/bug-report/bin/python ~/mypy_project_scope.py --config-file /mnt/pixery/ihsan/bugreport/mypy.ini /mnt/pixery/ihsan/bugreport/test.py
cwd: /mnt/pixery/ihsan/bugreport
> /mnt/pixery/miniconda3/envs/vscode-dev/bin/pydocstyle --config=/mnt/pixery/ihsan/bugreport/.pydocstyle /mnt/pixery/ihsan/bugreport/test.py
cwd: /mnt/pixery/ihsan/bugreport
##########Linting Output - pydocstyle##########
##########Linting Output - bandit##########
##########Linting Output - flake8##########
##########Linting Output - mypy##########
Success: no issues found in 1 source file
> /mnt/pixery/miniconda3/envs/pytorch/bin/python ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py -c "import jupyter"
> /mnt/pixery/miniconda3/envs/pytorch/bin/python ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py -c "import notebook"
> /mnt/pixery/miniconda3/envs/pytorch/bin/python ~/.vscode-server/extensions/ms-python.python-2020.9.114305/pythonFiles/pyvsc-run-isolated.py jupyter kernelspec --version

[Info  - 11:04:56 AM] Pylance language server 2020.10.0 (pyright d1f950ac) starting
[Info  - 11:04:56 AM] Server root directory: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist
[Info  - 11:04:56 AM] No configuration file found.
[Info  - 11:04:56 AM] Setting pythonPath for service "bugreport": "/mnt/pixery/miniconda3/envs/bug-report/bin/python"
Search paths found for configured python interpreter:
  /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8
  /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/lib-dynload
  /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages
[Error - 11:04:56 AM] stubPath /mnt/pixery/ihsan/bugreport/typings is not a valid directory.
[Info  - 11:04:56 AM] Assuming Python version 3.8
[Info  - 11:04:56 AM] Assuming Python platform Linux
[Info  - 11:04:56 AM] Searching for source files
[Info  - 11:04:56 AM] Found 1 source file
[Info  - 11:04:56 AM] Background analysis(1) root directory: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist
[Info  - 11:04:56 AM] Background analysis(1) started
Background analysis message: setConfigOptions
Background analysis message: setTrackedFiles
Background analysis message: markAllFilesDirty
Background analysis message: setFileOpened
Background analysis message: analyze
[FG] parsing: /mnt/pixery/ihsan/bugreport/test.py (35ms)
[FG] parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/2and3/builtins.pyi [fs read 5ms] (268ms)
[FG] binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/2and3/builtins.pyi (102ms)
[FG] binding: /mnt/pixery/ihsan/bugreport/test.py (2ms)
[BG(1)] analyzing: /mnt/pixery/ihsan/bugreport/test.py ...
[BG(1)]   parsing: /mnt/pixery/ihsan/bugreport/test.py (50ms)
[BG(1)]   parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/2and3/builtins.pyi [fs read 5ms] (271ms)
[BG(1)]   binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/2and3/builtins.pyi (94ms)
[BG(1)]   binding: /mnt/pixery/ihsan/bugreport/test.py (2ms)
[BG(1)]   checking: /mnt/pixery/ihsan/bugreport/test.py ...
[BG(1)]     parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/__init__.py [fs read 2ms] (64ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/__init__.py ...
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/__init__.py [fs read 1ms] (26ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/__init__.py ...
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/numeric.py [fs read 1ms] (102ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/numeric.py ...
[BG(1)]           parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/umath.py [fs read 0ms] (2ms)
[BG(1)]           binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/umath.py (2ms)
[BG(1)]           parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/numerictypes.py [fs read 0ms] (20ms)
[BG(1)]           binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/numerictypes.py (5ms)
[BG(1)]           parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/fromnumeric.py [fs read 0ms] (42ms)
[BG(1)]           binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/fromnumeric.py (8ms)
[BG(1)]           parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/arrayprint.py [fs read 0ms] (75ms)
[BG(1)]           binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/arrayprint.py (15ms)
[BG(1)]           parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/_asarray.py [fs read 0ms] (7ms)
[BG(1)]           binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/_asarray.py (1ms)
[BG(1)]           parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/_ufunc_config.py [fs read 0ms] (11ms)
[BG(1)]           binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/_ufunc_config.py (2ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/numeric.py (201ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/records.py [fs read 0ms] (24ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/records.py (7ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/memmap.py [fs read 0ms] (7ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/memmap.py (2ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/function_base.py [fs read 0ms] (11ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/function_base.py (4ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/machar.py [fs read 0ms] (21ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/machar.py (4ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/getlimits.py [fs read 0ms] (21ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/getlimits.py (6ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/shape_base.py [fs read 1ms] (14ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/shape_base.py (5ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/einsumfunc.py [fs read 0ms] (25ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/einsumfunc.py (16ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/__init__.py (475ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/__init__.py [fs read 0ms] (7ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/__init__.py ...
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/type_check.py [fs read 0ms] (9ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/type_check.py (3ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/index_tricks.py [fs read 0ms] (17ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/index_tricks.py (5ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/function_base.py [fs read 1ms] (73ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/function_base.py (30ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/nanfunctions.py [fs read 0ms] (23ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/nanfunctions.py (3ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/shape_base.py [fs read 0ms] (13ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/shape_base.py (14ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/stride_tricks.py [fs read 0ms] (2ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/stride_tricks.py (2ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/twodim_base.py [fs read 0ms] (7ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/twodim_base.py (5ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/ufunclike.py [fs read 0ms] (2ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/ufunclike.py (1ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/histograms.py [fs read 0ms] (9ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/histograms.py (6ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/polynomial.py [fs read 1ms] (24ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/polynomial.py (7ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/utils.py [fs read 1ms] (28ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/utils.py (6ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/arraysetops.py [fs read 0ms] (5ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/arraysetops.py (5ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/npyio.py [fs read 1ms] (59ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/npyio.py (23ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/financial.py [fs read 0ms] (11ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/financial.py (3ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/arraypad.py [fs read 0ms] (8ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/arraypad.py (8ms)
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/_version.py [fs read 0ms] (3ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/_version.py (1ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/lib/__init__.py (419ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/matrixlib/__init__.py [fs read 0ms] (1ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/matrixlib/__init__.py ...
[BG(1)]         parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/matrixlib/defmatrix.py [fs read 1ms] (12ms)
[BG(1)]         binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/matrixlib/defmatrix.py (6ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/matrixlib/__init__.py (28ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/__init__.py (962ms)
[BG(1)]     parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/__init__.py [fs read 0ms] (15ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/__init__.py (1ms)
[BG(1)]     parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/__init__.py [fs read 0ms] (5ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/__init__.py ...
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/waveforms.py [fs read 0ms] (6ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/waveforms.py (2ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/bsplines.py [fs read 1ms] (7ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/bsplines.py (6ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/filter_design.py [fs read 1ms] (61ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/filter_design.py (23ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/fir_filter_design.py [fs read 0ms] (20ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/fir_filter_design.py (7ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/ltisys.py [fs read 0ms] (39ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/ltisys.py (10ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/lti_conversion.py [fs read 0ms] (5ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/lti_conversion.py (6ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/signaltools.py [fs read 1ms] (65ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/signaltools.py (21ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/spectral.py [fs read 1ms] (11ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/spectral.py (9ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/wavelets.py [fs read 0ms] (18ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/wavelets.py (2ms)
[BG(1)]       parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/_peak_finding.py [fs read 0ms] (9ms)
[BG(1)]       binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/_peak_finding.py (3ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/scipy/signal/__init__.py (334ms)
[BG(1)]     parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/2and3/_typeshed/__init__.pyi [fs read 0ms] (2ms)
[BG(1)]     binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/2and3/_typeshed/__init__.pyi (1ms)
[BG(1)]     parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/third_party/2and3/typing_extensions.pyi [fs read 0ms] (1ms)
[BG(1)]     binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/third_party/2and3/typing_extensions.pyi (1ms)
[BG(1)]     parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/typing.pyi [fs read 1ms] (10ms)
[BG(1)]     binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/typing.pyi (12ms)
[BG(1)]     parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/functools.pyi [fs read 0ms] (1ms)
[BG(1)]     binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/functools.pyi (2ms)
[BG(1)]     parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/overrides.py [fs read 0ms] (3ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/overrides.py (1ms)
[BG(1)]     parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/abc.pyi [fs read 0ms] (0ms)
[BG(1)]     binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/abc.pyi (0ms)
[BG(1)]     parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/types.pyi [fs read 1ms] (11ms)
[BG(1)]     binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/types.pyi (2ms)
[BG(1)]     parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/ast.pyi [fs read 0ms] (3ms)
[BG(1)]     binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/ast.pyi ...
[BG(1)]       parsing: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/_ast.pyi [fs read 0ms] (3ms)
[BG(1)]       binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/_ast.pyi (2ms)
[BG(1)]     binding: /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2020.10.0/dist/typeshed-fallback/stdlib/3/ast.pyi (10ms)
[BG(1)]     parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/compat/_inspect.py [fs read 0ms] (3ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/compat/_inspect.py (4ms)
[BG(1)]     parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/multiarray.py [fs read 0ms] (7ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/core/multiarray.py (2ms)
[BG(1)]     parsing: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/random/__init__.py [fs read 0ms] (3ms)
[BG(1)]     binding: /mnt/pixery/miniconda3/envs/bug-report/lib/python3.8/site-packages/numpy/random/__init__.py (1ms)
[BG(1)]   checking: /mnt/pixery/ihsan/bugreport/test.py (1572ms)
[BG(1)] analyzing: /mnt/pixery/ihsan/bugreport/test.py (1991ms)
Background analysis message: resumeAnalysis
```

## Code Snippet / Additional information

```python
import numpy as np
from scipy import signal


def test(im, window_1d):
    triangular_1d = np.convolve(window_1d, window_1d)
    triangular_2d = np.outer(triangular_1d, triangular_1d)

    im_filtered = signal.convolve2d(im, triangular_2d, mode='same', boundary='symm')

    return im_filtered


if __name__ == '__main__':
    im = np.random.randn(40, 70).astype(np.float32)
    window_1d = np.ones(2) / 2

    im_filtered = test(im, window_1d)

    print(im_filtered)
```

**Comment by erictraut** [comment]
Thanks for reporting the problem. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.10.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020101-14-october-2020
